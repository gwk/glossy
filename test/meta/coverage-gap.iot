{
  'cmd': '$PROJ/iotest.py -no-times -coverage test-meta/$NAME.iot',
  'links' : {'test-meta'},
  'out_mode': 'match',
}