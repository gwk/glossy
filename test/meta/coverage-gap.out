test-meta/coverage-gap

################################################################
TESTS PASSED: 1.

----------------
Coverage Report:

__main__: test-meta/coverage-gap.py:
   1   #!/usr/bin/env python3
   2
   3   def f():
   4 !   return None

__main__: test-meta/coverage-gap.py: 4 lines; 2 trivial; 2 traceable; 1 covered; 0 relaxed; 0 ignored; 0 ignored but covered; 1 not covered.
